### DISCUSSION TITLE
ubuntu 에서 ~/.aws/config 파일 E212에러 발생

### DISCUSSION THREAD

**Discussion by peterjc85** [tedilabs/fastcampus-devops#28] [Q&A] [answered]
강의따라가다보면
~/aws/config 에 IAM KEY 복사하려는데, 
E212 에러가 발생합니다.

그리고 만약 WSL2의 우분투를 지우고 초기화 하고 싶으면 스토어에서 삭제 후 재설치 하면 되나요?

**Comment by peterjc85** [comment, marked as answer]
자답입니다.
aws configure로 key추가 후 파일생성 후 수정하였습니다.

**Reply by posquit0** [reply]
답변이 늦어 죄송합니다~
E212 에러는 Vim 에러를 뜻하는 듯 합니다. 이는 디렉토리가 생성되지 않은 상태로 디렉토리 내 파일을 작업한 경우인데, `~/.aws` 디렉토리가 없는 상태에서 `~/.aws/config` 파일을 저장하려고 하여 발생한 에러입니다.
추후에는 `mkdir -p ~/.aws` 명령어로 디렉토리 먼저 생성하시고 작업하실 수 있습니다.

감사합니다.
